doc/api/crypto.md

```diff
@@ -1804,6 +1804,16 @@ added: v15.6.0
 
 The issuer identification included in this certificate.
 
+### `x509.issuerCert`
```

Comment:
Given other API names such as `getPeerX509Certificate` and `getX509Certificate`, I have a slight preference not to abbreviate this name and to use `issuerCertificate` instead.